lib/watchErrorHandler.js

```diff
@@ -0,0 +1,7 @@
+module.exports = (message, status) =>  {
+  if(status === 0) {
+    console.log(message);
+  } else {
+    console.error( message.stack ? message.stack : String(message));
```

Comment:
As long as the output looks fine, I'm certainly down for consistency. I only brought it up as outputting objects in JS can be inconsistent depending on how its done.